frontend/src/app/wrapper/Wrapper.css

```diff
@@ -0,0 +1,133 @@
+.Wrapper {
+    min-height: 100%;
+    min-width: 100%;
+    display: flex; 
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.Header {
+    display: flex;
+    background-color:#FBFBFD;
+    height: 48px;
+    flex-direction: row;
+    align-self:stretch;
+    align-items:center;
+    justify-content: space-between;
+}
+
+.Header .logo {
+    flex-shrink: 0;
+    padding-left: 40px;
+    flex-basis: 25%;
+    justify-content: flex-start;
+}
+
+.Header .searchWrapper {
+    flex-shrink: 1;
+    flex-basis: 500px;
+}
+
+.Header .sign {
+    display: flex;
+    padding-right: 40px;
+    flex-shrink: 0;
+    flex-basis: 25%;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.Header .sign .button {
+    background-color: #111B47;
+    margin-left: 16px;
+    border: none;
+    color: white;
+    border-radius: 5px;
+    width: 120px;
+    height: 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+}
+
+.Content {
+    position: relative;
+    align-self: stretch;
+}
+
+.PlayerBar {
+    display: flex;
+    position: absolute;
```

Comment:
현재 스크롤을 내리면 playerbar가 움직이는 현상이 일어납니다.

![image](https://user-images.githubusercontent.com/77932817/140937929-ab141d25-9685-4992-ac24-49af492a470f.png)

```suggestion
    position: fixed;
```

이를 해결하려면 위와 같이 `position: fixed`로 바꾸면 된다고 합니다.